Feature/manager leaves 主管團隊假單

CarrieChien opened the pull request.

CarrieChien committed:
假別管理 coding style 調整

CarrieChien committed:
Merge branch 'master' into feature/types

wabowEno committed:
載入社交登入套件

wabowEno committed:
login頁面到目前為止的更新+debuger
終於套好了

TonyChiou committed:
coding style 修改差一點

CarrieChien committed:
假別管理 1.0

michaelhunag committed:
1. 目前為止的國定假日修改

michaelhunag committed:
Merge branch 'master' into feature/holiday

TonyChiou committed:
coding style修正 包括圖片上傳 validate驗證+中文化顯示 修改失敗帶回原本的值

michaelhunag committed:
1. 防呆 :新增 修改 如果有失敗的情況

CarrieChien committed:
假別管理 coding style 修正 2.0

CarrieChien committed:
假別管理 (if else foreach ) code review後 調整

CarrieChien committed:
假別管理 修正討論後問題；
(if else foreach 空行
& function 空行
& use 位置
& 變數寫法
& search用 hasColumn
& where 條件寫法)

TonyChiou committed:
code review之後的修改

michaelhunag committed:
1. code review 修正
2. error msh 修改
3. dashboard 實作

michaelhunag committed:
1. 修正上傳到 dashboard功能部份

michaelhunag committed:
1. dashboard 測試中

michaelhunag committed:
1. 資料從 DB 撈出

CarrieChien committed:
假別管理 error message 統一寫入error.blade.php

wabowEno committed:
修正Logout的轉址問題

TonyChiou committed:
補上變數宣告，員工管理完成

TonyChiou committed:
修改沒有連結沒有權限的route時候 會被redirect到 登入頁面

TonyChiou committed:
修改沒有連結沒有權限的route時候 會被redirect到 登入頁面

CarrieChien committed:
<Code Review>修正假別日期可以不用一定要填寫 & 麵包列route & 假別資料顯示

Commit:
Merge commit '68736847f9fbaa79459ffa35b502f627a7e79221' into develop

* commit '68736847f9fbaa79459ffa35b502f627a7e79221':
  補上變數宣告，員工管理完成
  code review之後的修改
  coding style修正 包括圖片上傳 validate驗證+中文化顯示 修改失敗帶回原本的值
  coding style 修改差一點

# Conflicts:
#	app/User.php
#	resources/views/layouts/head_css.blade.php
#	resources/views/layouts/header.blade.php
#	resources/views/users.blade.php
#	resources/views/users_form.blade.php
#	routes/web.php
#	storage/sqllog/history.sql

TonyChiou committed:
搜尋修正 確認無誤

TonyChiou committed:
Merge branch 'feature/employee' into develop

michaelhunag committed:
1. dashboard ajax
2. model 資料用 hasone 關聯

michaelhunag committed:
1. 刪除不用的code

Commit:
程式邏輯錯誤+coding style 調整

Commit:
程式邏輯錯誤+coding style 調整

Commit:
helper 寫法 : image 上傳調整為 helper 方式處理。

Commit:
helper 寫法 : image 上傳調整為 helper 方式處理。

michaelhunag committed:
1. 修正 controller name
2. 修正 function name

michaelhunag committed:
1. 修改 post csrf 寫法
2. 補上判斷 user 為空的情況

TonyChiou committed:
將index跟search走同一支route、排序補上href、側邊欄打開會顯示現在在哪、redirect用route不用to

michaelhunag committed:
1. 修正頁數 & 預設排序
2. html 頁修正

TonyChiou committed:
修正action寫法 直接抓route的name~

michaelhunag committed:
1. index 跟 search 走同一支
2. 左側 focus 補上
3. title 手指 補上

Commit:
Merge branch 'feature/user_login' into develop

Commit:
Merge branch 'feature/image_helper' of https://github.com/dioWabow/leave into feature/image_helper

* 'feature/image_helper' of https://github.com/dioWabow/leave:
  helper 寫法 : image 上傳調整為 helper 方式處理。

Commit:
Merge branch 'feature/user_login' of https://github.com/dioWabow/leave into feature/user_login

* 'feature/user_login' of https://github.com/dioWabow/leave:
  程式邏輯錯誤+coding style 調整
  修改沒有連結沒有權限的route時候 會被redirect到 登入頁面

Commit:
Merge branch 'feature/employee' into develop

* feature/employee:
  修正action寫法 直接抓route的name~
  將index跟search走同一支route、排序補上href、側邊欄打開會顯示現在在哪、redirect用route不用to

Commit:
1. 增加 BaseModel - 複寫原本 save 功能
2. 修正 login bug
3. 移除不需要的 merge 資料

michaelhunag committed:
1. 修正 dio code review 問題

Commit:
補上 nickname

TonyChiou committed:
修正當資料庫不存在資料時的判斷

michaelhunag committed:
1. 修正 變數名稱
2. 修正新增會出現預設日期

CarrieChien committed:
<假別管理><code review>後修正

michaelhunag committed:
1. 修正 function 同 tony

Commit:
Merge branch 'feature/employee' into develop

* feature/employee:
  修正當資料庫不存在資料時的判斷

Commit:
Merge branch 'feature/user_login' into develop

* feature/user_login:
  程式邏輯錯誤+coding style 調整
  修改沒有連結沒有權限的route時候 會被redirect到 登入頁面

michaelhunag committed:
1. 使用 BaseModel
2. 修正 all 用法
3. 修正 變數命名

Commit:
Merge branch 'feature/image_helper' into develop

* feature/image_helper:
  helper 寫法 : image 上傳調整為 helper 方式處理。

TonyChiou committed:
代理人跟團隊 BUG修正

Commit:
Merge branch 'hotfix/employee_fix' into develop

* hotfix/employee_fix:
  代理人跟團隊 BUG修正

# Conflicts:
#	app/Http/Controllers/UserController.php
#	storage/debugbar/.gitignore

Commit:
Merge branch 'feature/dash_board' into develop

* feature/dash_board:
  1. 修正 function 同 tony
  1. 修改 post csrf 寫法 2. 補上判斷 user 為空的情況
  1. 修正 controller name 2. 修正 function name
  1. dashboard ajax 2. model 資料用 hasone 關聯
  1. 資料從 DB 撈出
  1. dashboard 測試中

# Conflicts:
#	app/Team.php
#	app/UserTeam.php
#	resources/views/index.blade.php
#	routes/web.php
#	storage/debugbar/.gitignore

TonyChiou committed:
小錯誤修正

Commit:
Merge branch 'feature/holiday' into develop

* feature/holiday:
  1. 使用 BaseModel 2. 修正 all 用法 3. 修正 變數命名
  1. 修正 變數名稱 2. 修正新增會出現預設日期
  1. 修正 dio code review 問題
  1. index 跟 search 走同一支 2. 左側 focus 補上 3. title 手指 補上
  1. 修正頁數 & 預設排序 2. html 頁修正
  1. 刪除不用的code
  1. 修正上傳到 dashboard功能部份
  1. code review 修正 2. error msh 修改 3. dashboard 實作
  1. 防呆 :新增 修改 如果有失敗的情況
  1. 目前為止的國定假日修改

# Conflicts:
#	app/BaseModel.php
#	resources/lang/zh_tw/validation.php
#	resources/views/default.blade.php
#	resources/views/holidies.blade.php
#	resources/views/holidies_form.blade.php
#	resources/views/index.blade.php
#	resources/views/layouts/head_css.blade.php
#	routes/web.php
#	storage/sqllog/history.sql

Commit:
Revert "Merge branch 'feature/holiday' into develop"

This reverts commit 16e1c5c6e522aae5034b06b399f8bb6faa519b19, reversing
changes made to 2b4a3a8ca4e41fa519facd90b65fc6cc9d439f0c.

Commit:
no message

Commit:
Merge branch 'feature/holiday' into develop

* feature/holiday:
  no message

# Conflicts:
#	app/Http/Controllers/HolidayController.php

Commit:
Merge remote-tracking branch 'origin/hotfix/employee_fix2' into develop

* origin/hotfix/employee_fix2:
  小錯誤修正

Commit:
將麥可的 holiday 弄回來Ｔ＿＿Ｔ

CarrieChien committed:
<測試後修正>

Commit:
Merge branch 'feature/types' into develop

* feature/types:
  <測試後修正>
  <假別管理><code review>後修正
  <Code Review>修正假別日期可以不用一定要填寫 & 麵包列route & 假別資料顯示
  假別管理 error message 統一寫入error.blade.php
  假別管理 修正討論後問題； (if else foreach 空行 & function 空行 & use 位置 & 變數寫法 & search用 hasColumn & where 條件寫法)
  假別管理 (if else foreach ) code review後 調整
  假別管理 coding style 修正 2.0
  假別管理 1.0
  假別管理 coding style 調整

# Conflicts:
#	app/BaseModel.php
#	app/Leave.php
#	resources/lang/zh_tw/validation.php
#	resources/views/layouts/head_css.blade.php
#	resources/views/layouts/header.blade.php
#	resources/views/leave_type.blade.php
#	resources/views/leave_type_form.blade.php
#	routes/web.php
#	storage/debugbar/.gitignore
#	storage/sqllog/history.sql

Commit:
1. 增加 WebHelper
2. 拿掉 ImageHelper 測試 function

Commit:
1. 增加 WebHelper
2. 增加 UrlHelper
3. 調整 route
4. 調整 header 上的個人資料
5. 調整因為 route 變更後其他頁面的調整

Commit:
baseModel bug fix

wabowEno committed:
因為延遲太久+需要修改login
直接把舊版config開新分隻處理抓新develop再移植
更新config使用RollBack、ConfigHelper、前台頁面全部換成ConfigHelper
並且將原版Config移植過來

wabowEno committed:
code review更新
修正ImageHelper寫法增加指定filename功能

wabowEno committed:
更新image url不使用陣列[0]取得

wabowEno committed:
Merge branch 'feature/config_new' into develop

CarrieChien committed:
<主管團隊假單>
1. 主管級假單同一個controller
2. boss manager minimanager 使用同頁面

TonyChiou committed:
merge完成 待會繼續修改login

TonyChiou committed:
我要請假功能完成 測試正常

TonyChiou committed:
我要請假程式碼修正 + 刪減 目前測試正常

TonyChiou committed:
route修正為群組樣式

TonyChiou committed:
我要放假 code review後的修正

TonyChiou committed:
將誤刪的程式碼補回去

TonyChiou committed:
謀職假判斷 + coding style修正

wabowEno committed:
更新Auth取得目前權限功能

wabowEno committed:
修正註釋與小部分預設值以防未登入時出錯

CarrieChien committed:
<主管假單> merge Auth 權限

CarrieChien committed:
<主管假單>
請michael 協助顯示主管&小主管行事曆的部分

michaelhunag committed:
1. 團隊假單 - 行事曆

michaelhunag committed:
1. 合併function  到 siteController

CarrieChien committed:
<主管團隊假單>
假單邏輯已及寫法修正調整
1. 此邏輯包含 主管、BOSS、小主管

CarrieChien committed:
<主管團隊假單>
修正日期與搜尋結果錯誤

CarrieChien committed:
<主管團隊假單>
修正歷史紀錄顯示以子單日期為主

michaelhunag committed:
1. 修正 只抓通過資料
2. 修正 沒有團隊的顏色給預設值

CarrieChien committed:
<主管團隊假單>
1. 加入可從頁面直接審核通過假單
2. 頁面調整

CarrieChien committed:
<主管團隊假單>
頁面審核邏輯修正

TonyChiou commented:
1.controller postInset少了 fill 跟save
2.controller function getHistoryLeaveIdForToDay&getHistoryLeaveIdForSearch 裡面的tag_id 宣告位置修正
3.route命名修正
4.repson.php 改為 response.php(重要)
5.getLeavesIdByToDay($leave_id) 改為 getLeavesIdByDate($leave_id,$date) 這樣可以更多人一起共用function

CarrieChien committed:
<主管團隊假單>
修正如下：
1.controller postInset少了 fill 跟save
2.controller function getHistoryLeaveIdForToDay&getHistoryLeaveIdForSearch 裡面的tag_id 宣告位置修正
3.route命名修正
4.repson.php 改為 response.php(重要)
5.getLeavesIdByToDay($leave_id) 改為 getLeavesIdByDate($leave_id,$date) 這樣可以更多人一起共用function

CarrieChien committed:
<主管團隊假單>
merge 主管行事曆
